From 9102071645ac5ada994b323450fc206356353ecf Mon Sep 17 00:00:00 2001
From: Tobias <ruck.tobias@gmail.com>
Date: Tue, 26 Nov 2024 21:32:50 +0100
Subject: [PATCH] [doged] Rename dogecoin* executables to doge* (#85)

This is part of an effort to rebrand the node from Dogecash to doged, see #82.

Current-ABC adaptation (2026-10-14): Complete optional tool target names doge-qt, doge-wallet, doge-tx, doge-chainstate and doge-seeder across current CMake, packaging and test consumers. Keep upstream source filenames and internal libraries; preserve feature selection. Original intermediate rename commits: 23b5a7991f9fd36c56dfc96f464f0ae59b213c2d, 66a075ee1be0fb82b51b91fe9e6ffa001ce9bb1a, 0b4455fa7634dce5c8fb0d7d0cc211d64a5d85c5, 55bcf2cb993f52f8d038a13aa6fae7404f9c29f1. GUI/installer execution requires separate platform validation.

Original-Commit: 75dc738ffe920e6b99690fd8206071dda9c357cb

diff --git a/PackageOptions.cmake b/PackageOptions.cmake
index bba3240201a858ffe3619ad8818d1cdf9df904c6..e17737e0b8bda0e3b3241b4233cd0b0c2d94f611 100644
--- a/PackageOptions.cmake
+++ b/PackageOptions.cmake
@@ -32,7 +32,7 @@ set(CPACK_SOURCE_GENERATOR "TGZ")
 # CPack NSIS installer options
 Include(InstallationHelper)
 set(CPACK_NSIS_EXECUTABLES_DIRECTORY "${CMAKE_INSTALL_BINDIR}")
-set(_nsis_bitcoin_qt "bitcoin-qt.exe")
+set(_nsis_bitcoin_qt "doge-qt.exe")
 
 set(CPACK_NSIS_URL_INFO_ABOUT "${CPACK_PACKAGE_HOMEPAGE_URL}")
 set(CPACK_NSIS_CONTACT "${CPACK_PACKAGE_CONTACT}")
diff --git a/cmake/modules/NSIS.template.in b/cmake/modules/NSIS.template.in
index 44d455895568a16ee0c2ce441cdb427aa2b9cf0a..1702775d2d2dfa71fcd604fcd12539d8bc1806f6 100644
--- a/cmake/modules/NSIS.template.in
+++ b/cmake/modules/NSIS.template.in
@@ -10,7 +10,7 @@ Unicode true
 !define VERSION @CPACK_PACKAGE_VERSION@
 !define COMPANY "@CPACK_PACKAGE_VENDOR@"
 !define URL "@CPACK_NSIS_URL_INFO_ABOUT@"
-!define BITCOIN_GUI_NAME "bitcoin-qt.exe"
+!define BITCOIN_GUI_NAME "doge-qt.exe"
 
 # MUI Symbol Definitions
 !define MUI_ICON "@CPACK_NSIS_MUI_ICON@"
@@ -80,8 +80,8 @@ Section -Main SEC0000
     File /oname=COPYING.txt "@CPACK_RESOURCE_FILE_LICENSE@"
     File /oname=readme.txt "@CPACK_PACKAGE_DESCRIPTION_FILE@"
     SetOutPath $INSTDIR\daemon
-    File "@CPACK_TEMPORARY_INSTALL_DIRECTORY@/@CPACK_NSIS_EXECUTABLES_DIRECTORY@/bitcoind.exe"
-    File "@CPACK_TEMPORARY_INSTALL_DIRECTORY@/@CPACK_NSIS_EXECUTABLES_DIRECTORY@/bitcoin-cli.exe"
+    File "@CPACK_TEMPORARY_INSTALL_DIRECTORY@/@CPACK_NSIS_EXECUTABLES_DIRECTORY@/doged.exe"
+    File "@CPACK_TEMPORARY_INSTALL_DIRECTORY@/@CPACK_NSIS_EXECUTABLES_DIRECTORY@/doge-cli.exe"
     SetOutPath $INSTDIR\doc
     File /r "@CPACK_TEMPORARY_INSTALL_DIRECTORY@/doc\*.md"
     File /r /x /nonfatal bitcoin_logo_doxygen.png "@CPACK_TEMPORARY_INSTALL_DIRECTORY@/doc\*.png"
diff --git a/contrib/CMakeLists.txt b/contrib/CMakeLists.txt
index 9ee8d5d3e1f1c5c01c619ae793d13fff67c5b7b5..7621956ac8f6e08e956aca55e7cd525d9b636191 100644
--- a/contrib/CMakeLists.txt
+++ b/contrib/CMakeLists.txt
@@ -20,7 +20,7 @@ exclude_from_source_package(
 	"zmq/"
 
 	# Files
-	"bitcoin-qt.pro"
+	"doge-qt.pro"
 	"README.md"
 	"valgrind.supp"
 )
diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 9f42dcce5c0c27fc07469d97a69579f1cb29f887..58b72fa6c9dc852c298d101f202641112f1c8e07 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -17,11 +17,11 @@ endforeach()
 option(BUILD_WALLET "Activate the wallet functionality" ON)
 option(BUILD_ZMQ "Activate the ZeroMQ functionalities" ON)
 option(BUILD_CLI "Build doge-cli" ON)
-option(BUILD_TX "Build bitcoin-tx" ON)
-option(BUILD_QT "Build bitcoin-qt" ON)
-option(BUILD_SEEDER "Build bitcoin-seeder" ON)
+option(BUILD_TX "Build doge-tx" ON)
+option(BUILD_QT "Build doge-qt" ON)
+option(BUILD_SEEDER "Build doge-seeder" ON)
 option(BUILD_LIBBITCOINCONSENSUS "Build the bitcoinconsenus shared library" ON)
-option(BUILD_CHAINSTATE "Build bitcoin-chainstate" OFF)
+option(BUILD_CHAINSTATE "Build doge-chainstate" OFF)
 option(BUILD_IGUANA "Activate the Iguana debugger" ON)
 option(ENABLE_BIP70 "Enable BIP70 (payment protocol) support in GUI" ON)
 option(ENABLE_HARDENING "Harden the executables" ON)
@@ -749,19 +749,19 @@ if(BUILD_WALLET)
 	add_subdirectory(wallet)
 	target_link_libraries(server wallet)
 
-	# bitcoin-wallet
-	add_executable(bitcoin-wallet bitcoin-wallet.cpp)
-	generate_windows_version_info(bitcoin-wallet
+	# doge-wallet
+	add_executable(doge-wallet bitcoin-wallet.cpp)
+	generate_windows_version_info(doge-wallet
 		DESCRIPTION "CLI tool for ${PACKAGE_NAME} wallets"
 	)
 
-	target_link_libraries(bitcoin-wallet wallet-tool common util)
+	target_link_libraries(doge-wallet wallet-tool common util)
 
-	add_to_symbols_check(bitcoin-wallet)
-	add_to_security_check(bitcoin-wallet)
+	add_to_symbols_check(doge-wallet)
+	add_to_security_check(doge-wallet)
 
-	install_target(bitcoin-wallet)
-	install_manpages(bitcoin-wallet)
+	install_target(doge-wallet)
+	install_manpages(doge-wallet)
 else()
 	target_sources(server PRIVATE dummywallet.cpp)
 endif()
@@ -784,7 +784,7 @@ add_library(rpcclient
 
 target_link_libraries(rpcclient univalue util)
 
-# bitcoin-seeder
+# doge-seeder
 if(BUILD_SEEDER)
 	add_subdirectory(seeder)
 endif()
@@ -806,23 +806,23 @@ if(BUILD_CLI)
 	install_manpages(doge-cli)
 endif()
 
-# bitcoin-tx
+# doge-tx
 if(BUILD_TX)
-	add_executable(bitcoin-tx bitcoin-tx.cpp)
-	generate_windows_version_info(bitcoin-tx
+	add_executable(doge-tx bitcoin-tx.cpp)
+	generate_windows_version_info(doge-tx
 		DESCRIPTION "CLI Bitcoin transaction editor utility"
 	)
 
-	target_link_libraries(bitcoin-tx bitcoinconsensus)
+	target_link_libraries(doge-tx bitcoinconsensus)
 
-	add_to_symbols_check(bitcoin-tx)
-	add_to_security_check(bitcoin-tx)
+	add_to_symbols_check(doge-tx)
+	add_to_security_check(doge-tx)
 
-	install_target(bitcoin-tx)
-	install_manpages(bitcoin-tx)
+	install_target(doge-tx)
+	install_manpages(doge-tx)
 endif()
 
-# bitcoin-chainstate
+# doge-chainstate
 if(BUILD_CHAINSTATE)
 	# TODO: libbitcoinkernel is a work in progress consensus engine library, as more
 	#       and more modules are decoupled from the consensus engine, this list will
@@ -982,16 +982,16 @@ if(BUILD_CHAINSTATE)
 		link_windows_dependencies(bitcoinkernel)
 	endif()
 
-	add_executable(bitcoin-chainstate bitcoin-chainstate.cpp)
+	add_executable(doge-chainstate bitcoin-chainstate.cpp)
 
-	target_link_libraries(bitcoin-chainstate bitcoinkernel)
+	target_link_libraries(doge-chainstate bitcoinkernel)
 
-	generate_windows_version_info(bitcoin-chainstate
+	generate_windows_version_info(doge-chainstate
 		DESCRIPTION "CLI Datadir information utility (experimental)"
 	)
 
-	add_to_symbols_check(bitcoin-chainstate)
-	add_to_security_check(bitcoin-chainstate)
+	add_to_symbols_check(doge-chainstate)
+	add_to_security_check(doge-chainstate)
 endif()
 
 # doged
diff --git a/src/bitcoin-chainstate.cpp b/src/bitcoin-chainstate.cpp
index 22b2cd8164e7d478eae61151cbb6d0dd66a47989..fced3cc60a6c71117ec8cfa5ccde2a63802e5058 100644
--- a/src/bitcoin-chainstate.cpp
+++ b/src/bitcoin-chainstate.cpp
@@ -2,7 +2,7 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 //
-// The bitcoin-chainstate executable serves to surface the dependencies required
+// The doge-chainstate executable serves to surface the dependencies required
 // by a program wishing to use Bitcoin ABC's consensus engine as it is right
 // now.
 //
diff --git a/src/bitcoin-tx.cpp b/src/bitcoin-tx.cpp
index 9c1d2bc3a57ca493687a6514616d42b53e6d6b57..59574b32e2ba6695c5c27157629140b5d092cc75 100644
--- a/src/bitcoin-tx.cpp
+++ b/src/bitcoin-tx.cpp
@@ -138,7 +138,7 @@ static int AppInitRawTx(int argc, char *argv[]) {
 
     if (argc < 2 || HelpRequested(gArgs) || gArgs.IsArgSet("-version")) {
         // First part of help message is specific to this utility
-        std::string strUsage = PACKAGE_NAME " bitcoin-tx utility version " +
+        std::string strUsage = PACKAGE_NAME " doge-tx utility version " +
                                FormatFullVersion() + "\n";
 
         if (gArgs.IsArgSet("-version")) {
@@ -146,9 +146,9 @@ static int AppInitRawTx(int argc, char *argv[]) {
         } else {
             strUsage +=
                 "\n"
-                "Usage:  bitcoin-tx [options] <hex-tx> [commands]  Update "
+                "Usage:  doge-tx [options] <hex-tx> [commands]  Update "
                 "hex-encoded bitcoin transaction\n"
-                "or:     bitcoin-tx [options] -create [commands]   Create "
+                "or:     doge-tx [options] -create [commands]   Create "
                 "hex-encoded bitcoin transaction\n"
                 "\n";
             strUsage += gArgs.GetHelpMessage();
diff --git a/src/bitcoin-wallet.cpp b/src/bitcoin-wallet.cpp
index 4da24c2b02998e4e1ee470898a2d310214d39807..bc3e8325acd6988b68ecac49a531fb9ee2f64ee0 100644
--- a/src/bitcoin-wallet.cpp
+++ b/src/bitcoin-wallet.cpp
@@ -60,7 +60,7 @@ static bool WalletAppInit(int argc, char *argv[]) {
     }
     if (argc < 2 || HelpRequested(gArgs) || gArgs.IsArgSet("-version")) {
         std::string usage =
-            strprintf("%s bitcoin-wallet version", PACKAGE_NAME) + " " +
+            strprintf("%s doge-wallet version", PACKAGE_NAME) + " " +
             FormatFullVersion() + "\n";
 
         if (gArgs.IsArgSet("-version")) {
@@ -68,14 +68,14 @@ static bool WalletAppInit(int argc, char *argv[]) {
         } else {
             usage +=
                 "\n"
-                "bitcoin-wallet is an offline tool for creating and "
+                "doge-wallet is an offline tool for creating and "
                 "interacting with " PACKAGE_NAME " wallet files.\n"
-                "By default bitcoin-wallet will act on wallets in the default "
+                "By default doge-wallet will act on wallets in the default "
                 "mainnet wallet directory in the datadir.\n"
                 "To change the target wallet, use the -datadir, -wallet and "
                 "-testnet/-regtest arguments.\n\n"
                 "Usage:\n"
-                "  bitcoin-wallet [options] <command>\n";
+                "  doge-wallet [options] <command>\n";
             usage += "\n" + gArgs.GetHelpMessage();
         }
         tfm::format(std::cout, "%s", usage);
@@ -134,7 +134,7 @@ int main(int argc, char *argv[]) {
 
     if (method.empty()) {
         tfm::format(std::cerr,
-                    "No method provided. Run `bitcoin-wallet -help` for "
+                    "No method provided. Run `doge-wallet -help` for "
                     "valid methods.\n");
         return EXIT_FAILURE;
     }
diff --git a/src/qt/CMakeLists.txt b/src/qt/CMakeLists.txt
index 29a8daca049fd40c4ea42bca93ead2511a06a076..1f82a65e0182cd09d494c9fbea54af431c34ebc6 100644
--- a/src/qt/CMakeLists.txt
+++ b/src/qt/CMakeLists.txt
@@ -1,6 +1,6 @@
 # Copyright (c) 2018 The Bitcoin developers
 
-project(bitcoin-qt)
+project(doge-qt)
 
 include(BrewHelper)
 find_brew_prefix(QT5_PREFIX qt5)
@@ -311,7 +311,7 @@ if(BUILD_WALLET)
 	# Automoc option.
 	set(AUTOMOC_MOC_OPTIONS -DENABLE_WALLET=1)
 
-	# Add wallet functionality to bitcoin-qt
+	# Add wallet functionality to doge-qt
 	target_sources(bitcoin-qt-base
 		PRIVATE
 			addressbookpage.cpp
@@ -344,7 +344,7 @@ if(BUILD_WALLET)
 			walletview.cpp
 	)
 
-	# Add BIP70 functionality to bitcoin-qt
+	# Add BIP70 functionality to doge-qt
 	if(ENABLE_BIP70)
 		target_sources(bitcoin-qt-base
 			PRIVATE
@@ -361,23 +361,23 @@ if(BUILD_WALLET)
 endif()
 
 # The executable
-add_executable(bitcoin-qt WIN32 main.cpp)
+add_executable(doge-qt WIN32 main.cpp)
 include(WindowsVersionInfo)
-generate_windows_version_info(bitcoin-qt
+generate_windows_version_info(doge-qt
 	DESCRIPTION "GUI node for Bitcoin"
 	ICONS
 		"res/icons/bitcoin.ico"
 		"res/icons/bitcoin_testnet.ico"
 )
-target_link_libraries(bitcoin-qt bitcoin-qt-base)
+target_link_libraries(doge-qt bitcoin-qt-base)
 
 include(BinaryTest)
-add_to_symbols_check(bitcoin-qt)
-add_to_security_check(bitcoin-qt)
+add_to_symbols_check(doge-qt)
+add_to_security_check(doge-qt)
 
 include(InstallationHelper)
-install_target(bitcoin-qt)
-install_manpages(bitcoin-qt)
+install_target(doge-qt)
+install_manpages(doge-qt)
 
 if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 	set(BITCOINQT_BUNDLE_ICON "res/icons/bitcoin.icns")
@@ -396,7 +396,7 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 	set(EMPTY_LPROJ_FILE "${CMAKE_CURRENT_BINARY_DIR}/empty.lproj")
 	file(TOUCH "${EMPTY_LPROJ_FILE}")
 
-	target_sources(bitcoin-qt PRIVATE
+	target_sources(doge-qt PRIVATE
 		"${BITCOINQT_BUNDLE_ICON}"
 		"${INFO_PLIST_STRINGS_PATH}"
 		"${EMPTY_LPROJ_FILE}"
@@ -408,7 +408,7 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 	)
 
 	set(BITCOIN_QT_OSX_BUNDLE_NAME "BitcoinABC-Qt")
-	set_target_properties(bitcoin-qt PROPERTIES
+	set_target_properties(doge-qt PROPERTIES
 		MACOSX_BUNDLE ON
 		OUTPUT_NAME "${BITCOIN_QT_OSX_BUNDLE_NAME}"
 		MACOSX_BUNDLE_INFO_PLIST "${CMAKE_SOURCE_DIR}/share/qt/Info.plist.cmake.in"
@@ -447,11 +447,11 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 		OUTPUT
 			"${STRIPPED_BUNDLE}"
 		COMMAND
-			${CMAKE_COMMAND} -E copy_directory "$<TARGET_BUNDLE_DIR:bitcoin-qt>" "${STRIPPED_BUNDLE}"
+			${CMAKE_COMMAND} -E copy_directory "$<TARGET_BUNDLE_DIR:doge-qt>" "${STRIPPED_BUNDLE}"
 		COMMAND
 			${CMAKE_STRIP} "${STRIPPED_BUNDLE}/Contents/MacOS/${BITCOIN_QT_OSX_BUNDLE_NAME}"
 		DEPENDS
-			bitcoin-qt
+			doge-qt
 	)
 
 	set(QT_BASE_TRANSLATIONS
diff --git a/src/qt/test/CMakeLists.txt b/src/qt/test/CMakeLists.txt
index 82744d69e03f428180e2981236fb152b61fb5368..cc8df3689971d74552ecc71e89277463a58f7aad 100644
--- a/src/qt/test/CMakeLists.txt
+++ b/src/qt/test/CMakeLists.txt
@@ -1,12 +1,12 @@
 # Copyright (c) 2018 The Bitcoin developers
 
-project(test_bitcoin-qt)
+project(test_doge-qt)
 
 include(TestSuite)
-create_test_suite(bitcoin-qt)
-add_dependencies(check check-bitcoin-qt)
+create_test_suite(doge-qt)
+add_dependencies(check check-doge-qt)
 
-add_test_to_suite(bitcoin-qt test_bitcoin-qt
+add_test_to_suite(doge-qt test_doge-qt
 	apptests.cpp
 	bitcoinaddressvalidatortests.cpp
 	guiutiltests.cpp
@@ -17,12 +17,12 @@ add_test_to_suite(bitcoin-qt test_bitcoin-qt
 	util.cpp
 )
 
-target_link_libraries(test_bitcoin-qt testutil bitcoin-qt-base Qt5::Test)
+target_link_libraries(test_doge-qt testutil bitcoin-qt-base Qt5::Test)
 
 # Wallet
 if(BUILD_WALLET)
-	# Add wallet functionality to test_bitcoin-qt
-	target_sources(test_bitcoin-qt
+	# Add wallet functionality to test_doge-qt
+	target_sources(test_doge-qt
 		PRIVATE
 			addressbooktests.cpp
 			wallettests.cpp
@@ -30,11 +30,11 @@ if(BUILD_WALLET)
 	)
 
 	if(ENABLE_BIP70)
-		target_sources(test_bitcoin-qt
+		target_sources(test_doge-qt
 			PRIVATE
 				paymentservertests.cpp
 		)
 	endif()
 
-	target_link_libraries(test_bitcoin-qt wallet)
+	target_link_libraries(test_doge-qt wallet)
 endif()
diff --git a/src/seeder/CMakeLists.txt b/src/seeder/CMakeLists.txt
index 874b570cc40fe88a64684a206bd382018146228c..48f64571fdf61ab7ef6b9e5126921ed8bd0e5779 100644
--- a/src/seeder/CMakeLists.txt
+++ b/src/seeder/CMakeLists.txt
@@ -2,7 +2,7 @@
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
-project(bitcoin-seeder)
+project(doge-seeder)
 
 add_library(seeder-base
 	bitcoin.cpp
@@ -12,16 +12,16 @@ add_library(seeder-base
 )
 target_link_libraries(seeder-base server)
 
-add_executable(bitcoin-seeder
+add_executable(doge-seeder
 	main.cpp
 )
-target_link_libraries(bitcoin-seeder seeder-base)
+target_link_libraries(doge-seeder seeder-base)
 
 include(BinaryTest)
-add_to_symbols_check(bitcoin-seeder)
-add_to_security_check(bitcoin-seeder)
+add_to_symbols_check(doge-seeder)
+add_to_security_check(doge-seeder)
 
 include(InstallationHelper)
-install_target(bitcoin-seeder)
+install_target(doge-seeder)
 
 add_subdirectory(test)
diff --git a/test/CMakeLists.txt b/test/CMakeLists.txt
index 7ee01d728ff0736eded752d4c7b0a612cdb26669..666d1fe562879c85820abd9aac775655c132ee59 100644
--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -29,7 +29,7 @@ if(BUILD_CLI)
 	list(APPEND _TEST_TARGET_DEPENDS doge-cli)
 endif()
 if(BUILD_WALLET)
-	list(APPEND _TEST_TARGET_DEPENDS bitcoin-wallet)
+	list(APPEND _TEST_TARGET_DEPENDS doge-wallet)
 endif()
 if(BUILD_CHRONIK)
 	find_package(Protobuf 3.0.0 REQUIRED)
@@ -115,7 +115,7 @@ if(BUILD_TX)
 		CUSTOM_TARGET_ARGS
 			COMMENT "Test Bitcoin utilities..."
 			DEPENDS
-				bitcoin-tx
+				doge-tx
 				${CMAKE_CURRENT_BINARY_DIR}/util/bitcoin-util-test.py
 	)
 
diff --git a/test/functional/test_framework/test_framework.py b/test/functional/test_framework/test_framework.py
index c92a12ba10c7551a652d8e35b08f1f3eacda2d5a..e8465263aeaa2552626611327ad19ce80a6455b7 100644
--- a/test/functional/test_framework/test_framework.py
+++ b/test/functional/test_framework/test_framework.py
@@ -1118,9 +1118,9 @@ class BitcoinTestFramework(metaclass=BitcoinTestMetaClass):
             raise SkipTest("wallet has not been compiled.")
 
     def skip_if_no_wallet_tool(self):
-        """Skip the running test if bitcoin-wallet has not been compiled."""
+        """Skip the running test if doge-wallet has not been compiled."""
         if not self.is_wallet_tool_compiled():
-            raise SkipTest("bitcoin-wallet has not been compiled")
+            raise SkipTest("doge-wallet has not been compiled")
 
     def skip_if_no_cli(self):
         """Skip the running test if bitcoin-cli has not been compiled."""
@@ -1146,7 +1146,7 @@ class BitcoinTestFramework(metaclass=BitcoinTestMetaClass):
         return self.config["components"].getboolean("ENABLE_WALLET")
 
     def is_wallet_tool_compiled(self):
-        """Checks whether bitcoin-wallet was compiled."""
+        """Checks whether doge-wallet was compiled."""
         return self.config["components"].getboolean("ENABLE_WALLET_TOOL")
 
     def is_chronik_compiled(self):
diff --git a/test/functional/tool_wallet.py b/test/functional/tool_wallet.py
index 4009feee045d4001fa5b57d5fc7d4ea3632ebbb3..82d8496d3a9ffbfdd649656cdbd975bd219df308 100644
--- a/test/functional/tool_wallet.py
+++ b/test/functional/tool_wallet.py
@@ -1,7 +1,7 @@
 # Copyright (c) 2018-2019 The Bitcoin Core developers
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
-"""Test bitcoin-wallet."""
+"""Test doge-wallet."""
 
 import hashlib
 import os
@@ -28,7 +28,7 @@ class ToolWalletTest(BitcoinTestFramework):
     def bitcoin_wallet_process(self, *args):
         binary = (
             self.config["environment"]["BUILDDIR"]
-            + "/src/bitcoin-wallet"
+            + "/src/doge-wallet"
             + self.config["environment"]["EXEEXT"]
         )
         args = [f"-datadir={self.nodes[0].datadir}", f"-chain={self.chain}"] + list(
@@ -84,7 +84,7 @@ class ToolWalletTest(BitcoinTestFramework):
             "Testing that various invalid commands raise with specific error messages"
         )
         self.assert_raises_tool_error("Invalid command: foo", "foo")
-        # `bitcoin-wallet help` raises an error. Use `bitcoin-wallet -help`.
+        # `doge-wallet help` raises an error. Use `doge-wallet -help`.
         self.assert_raises_tool_error("Invalid command: help", "help")
         self.assert_raises_tool_error(
             "Error: two methods provided (info and create). Only one method should"
diff --git a/test/util/bitcoin-util-test.py b/test/util/bitcoin-util-test.py
index 4727ef468434b56460db8ed052c6d7b91c07d53c..092a485776232262157132e70a6d2e7c0bd6a036 100755
--- a/test/util/bitcoin-util-test.py
+++ b/test/util/bitcoin-util-test.py
@@ -176,7 +176,7 @@ def bctest(testDir, testObj, buildenv):
         want_error = testObj["error_txt"]
         # Compare error text
         # TODO: ideally, we'd compare the strings exactly and also assert
-        # That stderr is empty if no errors are expected. However, bitcoin-tx
+        # That stderr is empty if no errors are expected. However, doge-tx
         # emits DISPLAY errors when running as a windows application on
         # linux through wine. Just assert that the expected error text appears
         # somewhere in stderr.
diff --git a/test/util/data/bitcoin-util-test.json b/test/util/data/bitcoin-util-test.json
index f4dc1ecf173960509ddd494c0ca8008c7c4b1bc2..6105799137f022fee14c1feb6913c5f357f592f3 100644
--- a/test/util/data/bitcoin-util-test.json
+++ b/test/util/data/bitcoin-util-test.json
@@ -1,108 +1,108 @@
 [
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-create", "nversion=1"],
     "output_cmp": "blanktxv1.hex",
     "description": "Creates a blank v1 transaction"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-json","-create", "nversion=1"],
     "output_cmp": "blanktxv1.json",
     "description": "Creates a blank v1 transaction (output in json)"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-"],
     "input": "blanktxv2.hex",
     "output_cmp": "blanktxv2.hex",
-    "description": "Creates a blank transaction when nothing is piped into bitcoin-tx"
+    "description": "Creates a blank transaction when nothing is piped into doge-tx"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-json","-create"],
     "output_cmp": "blanktxv2.json",
     "description": "Creates a blank transaction (output in json)"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-json","-"],
     "input": "blanktxv2.hex",
     "output_cmp": "blanktxv2.json",
-    "description": "Creates a blank transaction when nothing is piped into bitcoin-tx (output in json)"
+    "description": "Creates a blank transaction when nothing is piped into doge-tx (output in json)"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-create", "nversion=1foo"],
     "return_code": 1,
     "error_txt": "error: Invalid TX version requested",
     "description": "Tests the check for invalid nversion value"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-", "delin=1"],
     "input": "tx394b54bb.hex",
     "output_cmp": "tt-delin1-out.hex",
     "description": "Deletes a single input from a transaction"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-json", "-", "delin=1"],
     "input": "tx394b54bb.hex",
     "output_cmp": "tt-delin1-out.json",
     "description": "Deletes a single input from a transaction (output in json)"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-", "delin=31"],
     "input": "tx394b54bb.hex",
     "return_code": 1,
     "error_txt": "error: Invalid TX input index '31'",
     "description": "Attempts to delete an input with a bad index from a transaction. Expected to fail."
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-", "delin=1foo"],
     "input": "tx394b54bb.hex",
     "return_code": 1,
     "error_txt": "error: Invalid TX input index",
     "description": "Tests the check for an invalid input index with delin"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-", "delout=1"],
     "input": "tx394b54bb.hex",
     "output_cmp": "tt-delout1-out.hex",
     "description": "Deletes a single output from a transaction"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-json", "-", "delout=1"],
     "input": "tx394b54bb.hex",
     "output_cmp": "tt-delout1-out.json",
     "description": "Deletes a single output from a transaction (output in json)"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-", "delout=2"],
     "input": "tx394b54bb.hex",
     "return_code": 1,
     "error_txt": "error: Invalid TX output index '2'",
     "description": "Attempts to delete an output with a bad index from a transaction. Expected to fail."
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-", "delout=1foo"],
     "input": "tx394b54bb.hex",
     "return_code": 1,
     "error_txt": "error: Invalid TX output index",
     "description": "Tests the check for an invalid output index with delout"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-", "locktime=317000"],
     "input": "tx394b54bb.hex",
     "output_cmp": "tt-locktime317000-out.hex",
     "description": "Adds an nlocktime to a transaction"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-json", "-", "locktime=317000"],
     "input": "tx394b54bb.hex",
     "output_cmp": "tt-locktime317000-out.json",
     "description": "Adds an nlocktime to a transaction (output in json)"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-create", "locktime=317000foo"],
     "return_code": 1,
     "error_txt": "error: Invalid TX locktime requested",
     "description": "Tests the check for invalid locktime value"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create",
      "in=Z897de6bd6027a475eadd57019d4e6872c396d0716c4875a5f1a6fcfdf385c1f:0"],
@@ -110,7 +110,7 @@
     "error_txt": "error: invalid TX input txid",
     "description": "Tests the check for an invalid txid invalid hex"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create",
      "in=5897de6bd6:0"],
@@ -118,7 +118,7 @@
     "error_txt": "error: invalid TX input txid",
     "description": "Tests the check for an invalid txid valid hex but too short"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create",
      "in=5897de6bd6027a475eadd57019d4e6872c396d0716c4875a5f1a6fcfdf385c1f12:0"],
@@ -126,7 +126,7 @@
     "error_txt": "error: invalid TX input txid",
     "description": "Tests the check for an invalid txid valid hex but too long"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create",
      "in=5897de6bd6027a475eadd57019d4e6872c396d0716c4875a5f1a6fcfdf385c1f:0x"],
@@ -134,7 +134,7 @@
     "error_txt": "error: invalid TX input vout",
     "description": "Tests the check for an invalid vout value when adding an input"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create",
      "outaddr=1"],
@@ -142,7 +142,7 @@
     "error_txt": "error: TX output missing or too many separators",
     "description": "Malformed outaddr argument (no address specified). Expected to fail."
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create",
      "outaddr=1:D82zqZA4KqaTPmsGegJ1ACoTXiSJ21NcZd:garbage"],
@@ -150,7 +150,7 @@
     "error_txt": "error: TX output missing or too many separators",
     "description": "Malformed outaddr argument (too many separators). Expected to fail."
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create",
      "outpubkey=0"],
@@ -158,7 +158,7 @@
     "error_txt": "error: TX output missing or too many separators",
     "description": "Malformed outpubkey argument (no pubkey specified). Expected to fail."
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create",
      "outpubkey=0:02a5613bd857b7048924264d1e70e08fb2a7e6527d32b7ab1bb993ac59964ff397:W:non53nse"],
@@ -166,7 +166,7 @@
     "error_txt": "error: TX output missing or too many separators",
     "description": "Malformed outpubkey argument (too many separators). Expected to fail."
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create",
      "in=5897de6bd6027a475eadd57019d4e6872c396d0716c4875a5f1a6fcfdf385c1f:0",
@@ -177,7 +177,7 @@
     "output_cmp": "txcreate1.hex",
     "description": "Creates a new transaction with three inputs and two outputs"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-json",
      "-create",
@@ -189,87 +189,87 @@
     "output_cmp": "txcreate1.json",
     "description": "Creates a new transaction with three inputs and two outputs (output in json)"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-create", "outscript=0:"],
     "output_cmp": "txcreate2.hex",
     "description": "Creates a new transaction with a single empty output script"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-json", "-create", "outscript=0:"],
     "output_cmp": "txcreate2.json",
     "description": "Creates a new transaction with a single empty output script (output in json)"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["02000000000100000000000000000000000000"],
     "output_cmp": "txcreate2.hex",
     "description": "Parses a transaction with no inputs and a single output script"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-json", "02000000000100000000000000000000000000"],
     "output_cmp": "txcreate2.json",
     "description": "Parses a transaction with no inputs and a single output script (output in json)"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-create", "outscript=0:123badscript"],
     "return_code": 1,
     "error_txt": "error: script parse error: unknown opcode 123badscript",
     "description": "Create a new transaction with an invalid output script"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-create", "outscript=0:OP_DROP", "nversion=1"],
     "output_cmp": "txcreatescript1.hex",
     "description": "Create a new transaction with a single output script (OP_DROP)"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-json", "-create", "outscript=0:OP_DROP", "nversion=1"],
     "output_cmp": "txcreatescript1.json",
     "description": "Create a new transaction with a single output script (OP_DROP) (output as json)"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-create", "outscript=0:OP_DROP:S", "nversion=1"],
     "output_cmp": "txcreatescript2.hex",
     "description": "Create a new transaction with a single output script (OP_DROP) in a P2SH"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-json", "-create", "outscript=0:OP_DROP:S", "nversion=1"],
     "output_cmp": "txcreatescript2.json",
     "description": "Create a new transaction with a single output script (OP_DROP) in a P2SH (output as json)"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-create", "outscript=0:999999999999999999999999999999"],
     "return_code": 1,
     "error_txt": "error: script parse error: decimal numeric value only allowed in the range -0x7FFFFFFFFFFFFFFF...0x7FFFFFFFFFFFFFFF",
     "description": "Try to parse an output script with a decimal number above the allowed range"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-create", "outscript=0:9223372036854775808"],
     "return_code": 1,
     "error_txt": "error: script parse error: decimal numeric value only allowed in the range -0x7FFFFFFFFFFFFFFF...0x7FFFFFFFFFFFFFFF",
     "description": "Try to parse an output script with a decimal number just above the allowed range"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-create", "outscript=0:9223372036854775807"],
     "output_cmp": "txcreatescript5.hex",
     "description": "Try to parse an output script with a decimal number at the upper limit of the allowed range"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-create", "outscript=0:-9223372036854775808"],
     "return_code": 1,
     "error_txt": "error: script parse error: decimal numeric value only allowed in the range -0x7FFFFFFFFFFFFFFF...0x7FFFFFFFFFFFFFFF",
     "description": "Try to parse an output script with a decimal number below the allowed range"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-create", "outscript=0:-9223372036854775808"],
     "return_code": 1,
     "error_txt": "error: script parse error: decimal numeric value only allowed in the range -0x7FFFFFFFFFFFFFFF...0x7FFFFFFFFFFFFFFF",
     "description": "Try to parse an output script with a decimal number just below the allowed range"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-create", "outscript=0:-9223372036854775807"],
     "output_cmp": "txcreatescript6.hex",
     "description": "Try to parse an output script with a decimal number at the lower limit of the allowed range"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create", "nversion=1",
      "in=4d49a71ec9da436f71ec4ee231d04f292a29cd316f598bb7068feccabdc59485:0",
@@ -280,7 +280,7 @@
     "output_cmp": "txcreatesignv1.hex",
     "description": "Creates a new v1 transaction with a single input and a single output, and then signs the transaction"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-json",
      "-create", "nversion=1",
@@ -292,7 +292,7 @@
     "output_cmp": "txcreatesignv1.json",
     "description": "Creates a new v1 transaction with a single input and a single output, and then signs the transaction (output in json)"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create",
       "in=4d49a71ec9da436f71ec4ee231d04f292a29cd316f598bb7068feccabdc59485:0",
@@ -303,7 +303,7 @@
     "output_cmp": "txcreatesignv2.hex",
     "description": "Creates a new transaction with a single input and a single output, and then signs the transaction"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create",
       "in=4d49a71ec9da436f71ec4ee231d04f292a29cd316f598bb7068feccabdc59485:0",
@@ -315,7 +315,7 @@
     "error_txt": "error: prevtxs internal object typecheck fail",
     "description": "Tests the check for invalid vout index in prevtxs for sign"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create",
       "in=4d49a71ec9da436f71ec4ee231d04f292a29cd316f598bb7068feccabdc59485:0",
@@ -327,7 +327,7 @@
     "error_txt": "error: txid must be hexadecimal string (not 'Zd49a71ec9da436f71ec4ee231d04f292a29cd316f598bb7068feccabdc59412')",
     "description": "Tests the check for invalid txid due to invalid hex"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create",
       "in=4d49a71ec9da436f71ec4ee231d04f292a29cd316f598bb7068feccabdc59485:0",
@@ -339,7 +339,7 @@
     "error_txt": "error: txid must be hexadecimal string (not '4d49a71ec9da436f71ec4ee231d04f292a29cd316f598bb7068feccabdc594')",
     "description": "Tests the check for invalid txid valid hex, but too short"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create",
       "in=4d49a71ec9da436f71ec4ee231d04f292a29cd316f598bb7068feccabdc59485:0",
@@ -351,19 +351,19 @@
     "error_txt": "error: txid must be hexadecimal string (not '4d49a71ec9da436f71ec4ee231d04f292a29cd316f598bb7068feccabdc5948512')",
     "description": "Tests the check for invalid txid valid hex, but too long"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create", "outpubkey=0:02a5613bd857b7048924264d1e70e08fb2a7e6527d32b7ab1bb993ac59964ff397", "nversion=1"],
     "output_cmp": "txcreateoutpubkey1.hex",
     "description": "Creates a new transaction with a single pay-to-pubkey output"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-json", "-create", "outpubkey=0:02a5613bd857b7048924264d1e70e08fb2a7e6527d32b7ab1bb993ac59964ff397", "nversion=1"],
     "output_cmp": "txcreateoutpubkey1.json",
     "description": "Creates a new transaction with a single pay-to-pubkey output (output as json)"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create",
      "in=5897de6bd6027a475eadd57019d4e6872c396d0716c4875a5f1a6fcfdf385c1f:0",
@@ -372,7 +372,7 @@
     "error_txt": "error: invalid TX output data",
     "description": "Attempts to create a new transaction with one input and an output with malformed hex data. Expected to fail"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create",
      "in=5897de6bd6027a475eadd57019d4e6872c396d0716c4875a5f1a6fcfdf385c1f:0",
@@ -381,7 +381,7 @@
     "error_txt": "error: invalid TX output data",
     "description": "Attempts to create a new transaction with one input and an output with no value and malformed hex data. Expected to fail"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create",
      "in=5897de6bd6027a475eadd57019d4e6872c396d0716c4875a5f1a6fcfdf385c1f:0",
@@ -390,7 +390,7 @@
     "output_cmp": "txcreatedata1.hex",
     "description": "Creates a new transaction with one input, one address output and one data output"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-json",
      "-create", "nversion=1",
@@ -400,7 +400,7 @@
     "output_cmp": "txcreatedata1.json",
     "description": "Creates a new v1 transaction with one input, one address output and one data output (output in json)"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create",
      "in=5897de6bd6027a475eadd57019d4e6872c396d0716c4875a5f1a6fcfdf385c1f:0",
@@ -409,7 +409,7 @@
     "output_cmp": "txcreatedata2.hex",
     "description": "Creates a new transaction with one input, one address output and one data (zero value) output"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-json",
      "-create",
@@ -419,7 +419,7 @@
     "output_cmp": "txcreatedata2.json",
     "description": "Creates a new transaction with one input, one address output and one data (zero value) output (output in json)"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-create",
      "in=5897de6bd6027a475eadd57019d4e6872c396d0716c4875a5f1a6fcfdf385c1f:0:4294967293",
@@ -427,7 +427,7 @@
     "output_cmp": "txcreatedata_seq0.hex",
     "description": "Creates a new transaction with one input with sequence number and one address output"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-json",
      "-create",
@@ -436,14 +436,14 @@
     "output_cmp": "txcreatedata_seq0.json",
     "description": "Creates a new transaction with one input with sequence number and one address output (output in json)"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["01000000011f5c38dfcf6f1a5f5a87c416076d392c87e6d41970d5ad5e477a02d66bde97580000000000fdffffff0180a81201000000001976a9141fc11f39be1729bf973a7ab6a615ca4729d6457488ac00000000",
      "in=5897de6bd6027a475eadd57019d4e6872c396d0716c4875a5f1a6fcfdf385c1f:0:1"],
     "output_cmp": "txcreatedata_seq1.hex",
     "description": "Adds a new input with sequence number to a transaction"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args":
     ["-json",
      "01000000011f5c38dfcf6f1a5f5a87c416076d392c87e6d41970d5ad5e477a02d66bde97580000000000fdffffff0180a81201000000001976a9141fc11f39be1729bf973a7ab6a615ca4729d6457488ac00000000",
@@ -451,22 +451,22 @@
     "output_cmp": "txcreatedata_seq1.json",
     "description": "Adds a new input with sequence number to a transaction (output in json)"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-create", "outmultisig=1000000:2:3:02a5613bd857b7048924264d1e70e08fb2a7e6527d32b7ab1bb993ac59964ff397:021ac43c7ff740014c3b33737ede99c967e4764553d1b2b83db77c83b8715fa72d:02df2089105c77f266fa11a9d33f05c735234075f2e8780824c6b709415f9fb485", "nversion=1"],
     "output_cmp": "txcreatemultisig1.hex",
     "description": "Creates a new transaction with a single 2-of-3 multisig output"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-json", "-create", "outmultisig=1000000:2:3:02a5613bd857b7048924264d1e70e08fb2a7e6527d32b7ab1bb993ac59964ff397:021ac43c7ff740014c3b33737ede99c967e4764553d1b2b83db77c83b8715fa72d:02df2089105c77f266fa11a9d33f05c735234075f2e8780824c6b709415f9fb485", "nversion=1"],
     "output_cmp": "txcreatemultisig1.json",
     "description": "Creates a new transaction with a single 2-of-3 multisig output (output in json)"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-create", "outmultisig=1000000:2:3:02a5613bd857b7048924264d1e70e08fb2a7e6527d32b7ab1bb993ac59964ff397:021ac43c7ff740014c3b33737ede99c967e4764553d1b2b83db77c83b8715fa72d:02df2089105c77f266fa11a9d33f05c735234075f2e8780824c6b709415f9fb485:S", "nversion=1"],
     "output_cmp": "txcreatemultisig2.hex",
     "description": "Creates a new transaction with a single 2-of-3 multisig in a P2SH output"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./doge-tx",
     "args": ["-json", "-create", "outmultisig=1000000:2:3:02a5613bd857b7048924264d1e70e08fb2a7e6527d32b7ab1bb993ac59964ff397:021ac43c7ff740014c3b33737ede99c967e4764553d1b2b83db77c83b8715fa72d:02df2089105c77f266fa11a9d33f05c735234075f2e8780824c6b709415f9fb485:S", "nversion=1"],
     "output_cmp": "txcreatemultisig2.json",
     "description": "Creates a new transaction with a single 2-of-3 multisig in a P2SH output (output in json)"
